packages/flutter_tools/templates/plugin/windows.tmpl/pluginClassSnakeCase.h.tmpl

```diff
@@ -20,7 +20,6 @@ class {{pluginClass}} : public flutter::Plugin {
   {{pluginClass}}(const {{pluginClass}}&) = delete;
   {{pluginClass}}& operator=(const {{pluginClass}}&) = delete;
 
- private:
```

Comment:
This makes HandleMethodCall public, for the purposes of testing. We could use a friend, but that requires a test fixture, thus making the tests more complicated, and I don't see any real downside to this being public since the class itself isn't actually exposed to anything in normal use. (And on macOS/iOS, this corresponding method is a public method on the FlutterPlugin interface, so it's public for all plugins on those platforms.)